values.yaml

```diff
@@ -255,7 +255,51 @@ jobScheduler:
   service:
     annotations: {}
     type: ClusterIP
-
+staticContent:
+  image: actian/integration-manager-static-content:2.0.5.270
```

Comment:
Circle back to this tag once we have a valid tag created for integration-manager-static-content in dockerhub